code/src/cljs/sixsq/nuvla/ui/cimi/views.cljs

```diff
@@ -247,8 +247,7 @@
          [DocumentationButton]
 
          [ui/FormGroup
-          [ui/FormField
-           [cloud-entry-point-title]]]
+          [cloud-entry-point-title]]
```

Comment:
Now it's not aligned on smaller screens:
<img width="308" alt="image" src="https://user-images.githubusercontent.com/10104888/214867606-10844ab2-1e6c-45f9-b16b-0a0c9a9f8813.png">